homeassistant/components/russound_rio/config_flow.py

```diff
@@ -0,0 +1,117 @@
+"""Config flow to configure russound_rio component."""
+
+from __future__ import annotations
+
+import asyncio
+import logging
+from typing import Any
+
+from aiorussound import Russound
+import voluptuous as vol
+
+from homeassistant.config_entries import ConfigFlow, ConfigFlowResult
+from homeassistant.const import CONF_HOST, CONF_NAME, CONF_PORT
+from homeassistant.helpers import config_validation as cv
+
+from .const import DOMAIN, RUSSOUND_RIO_EXCEPTIONS, NoPrimaryControllerException
+
+DATA_SCHEMA = vol.Schema(
+    {
+        vol.Required(CONF_HOST): cv.string,
+        vol.Required(CONF_NAME): cv.string,
```

Comment:
Only helpers are allowed to have a settable name. Config flow intergations should set a name that is user recognizable, but not ask the user

Reply:
I based this off of the existing YAML config for the integration. I have no issue just removing the name entry and using the Russound type (model) as the title.